lib/configure.js

```diff
@@ -479,19 +479,32 @@ PythonFinder.prototype = {
   },
 
   failNoPython: function failNoPython () {
-    var errmsg =
-        'Can\'t find Python executable "' + this.python +
-        '", you can set the PYTHON env variable.'
-    this.callback(new Error(errmsg))
+    const err = new Error(
+      '\n******************************************************************\n' +
+      `node-gyp can't use "${this.python}",\n` +
+      'It is recommended that you install python 2.7, set the PYTHON env,\n' +
+      'or use the --python switch to point to a Python >= v2.6.0 & < 3.0.0.\n' +
```

Comment:
Should this be 2.7.0 instead of 2.6.0 at this point?  [Python 2.6 has been EOL for 5 years](https://docs.python.org/devguide/index.html#branchstatus).